activesupport/lib/active_support/time_with_zone.rb

```diff
@@ -103,7 +103,7 @@ def dst?
     #   Time.zone = 'Eastern Time (US & Canada)'    # => 'Eastern Time (US & Canada)'
     #   Time.zone.now.utc?                          # => false
     def utc?
-      zone == "UTC" || zone == "UCT"
+      zone == "UTC" || zone == "GMT"
```

Comment:
```suggestion
      zone == "UTC" || zone == "GMT" || zone == "UCT"
```

I don't think it's worth not checking for `UCT`, that introduce backward compatibility concerns, so might as well keep it, it's not an expensive check.